logging/loki/variables.tf

```diff
@@ -11,13 +22,39 @@ variable "cluster_name" {
   description = "A name of the Amazon EKS cluster"
 }
 
-variable "config_path" {
-  description = "location of the kubeconfig file"
-  default     = "~/.kube/config"
+variable "grafana_loki_password" {
+  type        = string
+  description = "Password for grafana admin"
+  default     = ""
 }
 
 variable "domains" {
   type        = list(string)
   default     = []
   description = "A list of domains to use for ingresses"
 }
+
+variable "namespace" {
+  type        = string
+  default     = ""
+  description = "A name of the existing namespace"
+}
+
+variable "namespace_name" {
+  type        = string
+  default     = "logging"
+  description = "A name of namespace for creating"
+}
+
+#pumped chart version
+variable "chart_version" {
+  type        = string
+  description = "A Helm Chart version"
+  default     = "2.0.0"
```

Comment:
last version is 2.3.1 - can you bump version at latest and check that it's work?